{
    "affected": [
        {
            "package": {
                "ecosystem": "photon:2.0",
                "name": "PyYAML",
                "purl": "pkg:rpm/vmware/PyYAML?distro=photon-2"
            },
            "ranges": {
                "events": [
                    {
                        "introduced": "0"
                    },
                    {
                        "fixed": "3.12-5.ph2"
                    }
                ],
                "type": "ECOSYSTEM"
            }
        },
        {
            "package": {
                "ecosystem": "Photon OS:2.0",
                "name": "PyYAML",
                "purl": "pkg:rpm/vmware/PyYAML?distro=photon-2"
            },
            "ranges": {
                "events": [
                    {
                        "introduced": "0"
                    },
                    {
                        "fixed": "3.12-5.ph2"
                    }
                ],
                "type": "ECOSYSTEM"
            }
        }
    ],
    "id": "PHSA-2020-0217",
    "modified": "2023-05-07T05:25:00Z",
    "published": "2020-03-08T00:00:00Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://github.com/vmware/photon/wiki/Security-Update-2.0-217"
        }
    ],
    "related": [
        "CVE-2019-20477"
    ]
}